src/Controller/Component/CrudComponent.php

```diff
@@ -547,7 +547,7 @@ public function defaults($type, $name, $config = null)
                 $this->config(sprintf('%s.%s', $type, $realName), $config);
             }
 
-            return;
+            return null;
```

Comment:
> But I'm against discussions on semantics where there is an IDE or "upstream" conceptual idea on what's corrects or not.

:+1:  I too don't care for unnecessary code churn unless the changes brings actual value.